tests/storage/test_pluggable.py

```diff
@@ -15,6 +15,9 @@ def get(self, key):
             return None
         return self._keys[key]
 
+    def get_many(self, keys):
+        return [self.get(key) for key in keys]
+
     def set(self, key, value):
         self._keys[key] = value
 
```

Comment:
Check `increment` and `decrement` methods, to return the updated value.
Example:
```
    def decrement(self, key, value):
        if key not in self._keys:
            self._keys[key] = 0
        self._keys[key]-= value
        return self._keys[key]
```

Reply:
fixed